docs/guides/js-waku/quick-start.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Quick Start
+---
+
+This guide provides quick steps to start using the `js-waku` SDK by setting up a Waku node and sending messages using the [Relay protocol](/overview/concepts/protocols#relay). Please refer to the [installation guide](/guides/js-waku/#installation) for steps on adding `js-waku` to your project.
+
+## Create a Relay Node
+
+Use the `createRelayNode()` function to create a relay node and interact with the Waku Network:
+
+```js
+import { createRelayNode } from "@waku/sdk";
+
+// Create and start a relay node
+const node = await createRelayNode({ defaultBootstrap: true });
+await node.start();
+
+// Use the stop() function to stop a running node
+// await node.stop();
+```
+
+:::info
+The `defaultBootstrap` option bootstraps your node using [pre-defined Waku nodes](/overview/concepts/static-peers).
+:::
+
+## Connect to Remote Peers
+
+Use the `waitForRemotePeer()` function to wait for the node to connect with peers on the Waku Network:
+
+```js
+import { waitForRemotePeer } from "@waku/sdk";
+
+// Wait for a successful peer connection
+await waitForRemotePeer(node);
+```
+
+The `protocols` option allows you to specify the [protocols](/overview/concepts/protocols) that the remote peers should have enabled:
+
+```js
+import { waitForRemotePeer, Protocols } from "@waku/sdk";
+
+// Wait for peer connections with specific protocols
+await waitForRemotePeer(node, [
+	Protocols.Relay,
+	Protocols.Store,
+	Protocols.LightPush,
+	Protocols.Filter,
+]);
+```
+
+## Choose a Content Topic
+
+[Choose a content topic](/overview/concepts/content-topics) for your application and create an `encoder` for [message encryption](https://rfc.vac.dev/spec/26/):
+
+```js
+import { createEncoder } from "@waku/sdk";
+
+// Choose a content topic
+const contentTopic = "/quick-start/1/message/proto";
+
+// Create a message encoder
+const encoder = createEncoder({ contentTopic });
```

Comment:
On the website, it shows the `ephemeral: false` option.with an info box to say that `true` means it is not stored.

I suggest instead:
don't use `ephemeral` in the first code block

in the info box, do show usage of `createEncoder` with `ephemeral: true`